src/protagonist/API/Features/DeliveryChannels/Requests/CreateDeliveryChannelPolicy.cs

```diff
@@ -0,0 +1,67 @@
+﻿using API.Features.DeliveryChannels.Validation;
+using API.Infrastructure.Requests;
+using DLCS.Core;
+using DLCS.Model.Policies;
+using DLCS.Repository;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace API.Features.DeliveryChannels.Requests;
+
+public class CreateDeliveryChannelPolicy : IRequest<ModifyEntityResult<DeliveryChannelPolicy>>
+{
+    public int CustomerId { get; }
+    
+    public DeliveryChannelPolicy DeliveryChannelPolicy { get; }
+    
+    public CreateDeliveryChannelPolicy(int customerId, DeliveryChannelPolicy deliveryChannelPolicy)
+    {
+        CustomerId = customerId;
+        DeliveryChannelPolicy = deliveryChannelPolicy;
+    }
+}
+
+public class CreateDeliveryChannelPolicyHandler : IRequestHandler<CreateDeliveryChannelPolicy, ModifyEntityResult<DeliveryChannelPolicy>>
+{
+    private readonly DlcsContext dbContext;
+    
+    public CreateDeliveryChannelPolicyHandler(DlcsContext dbContext, DeliveryChannelPolicyDataValidator policyDataValidator)
+    {
+        this.dbContext = dbContext;
+    }
+    
+    public async Task<ModifyEntityResult<DeliveryChannelPolicy>> Handle(CreateDeliveryChannelPolicy request, CancellationToken cancellationToken)
+    {
+        var nameInUse = await dbContext.DeliveryChannelPolicies.AnyAsync(p => 
+            p.Customer == request.CustomerId &&
+            p.Channel == request.DeliveryChannelPolicy.Channel &&
+            p.Name == request.DeliveryChannelPolicy.Name,
+            cancellationToken);
+        
+        if (nameInUse)
```

Comment:
Is this necessary? The `DeliveryChannelPolicies` object has a unique constraint on customer/channel/name so the DB would throw an exception if the name was in use - would save a DB trip and you could catch the appropriate exception to handle return a 409.